apps/web-remix/app/components/chat/chat.reducer.ts

```
@@ -84,7 +92,7 @@ export const chatReducer = (
       return { ...state, status: 'generating' };
 
     case 'STATUS_CHANGE':
-      if (Object.keys(state.outputsStatus).includes(action.payload.blockName)) {
+      if (state.outputsStatus.hasOwnProperty(action.payload.blockName)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)